exonum-java-binding/core/pom.xml

```diff
@@ -209,6 +209,24 @@
             </goals>
           </execution>
 
+          <execution>
+            <id>build-ejb-app</id>
```

Comment:
Added special `build-ejb-app` execution. It builds only EJB App (without integration tests)